schematools/db.py

```diff
@@ -139,6 +150,8 @@ def _serialize(obj, camelize=True):
 
 
 def _extract_names(properties):
+    result = {}
     for prop in properties:
         name = prop.pop("name").replace("_", " ")
-        yield {name: prop}
+        result[name] = prop
+    return result
```

Comment:
evt zelfs return `{prop.pop("name").replace("_", " "): prop for prop in properties}`

Reply:
Check, als het toch geen generator meer is, is dit beter.